puppet/puppet.go

```diff
@@ -26,7 +26,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/purpleidea/mgmt/yamlgraph"
+	yamlgraph "github.com/purpleidea/mgmt/yamlgraph2"
```

Comment:
Nack on this one line at least for now. Let's keep things stable until we deprecate yamlv1 and make sure felix is okay with the change first since his puppet-mgmtgraph depends on this format.